CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.0.0-beta8 (in progress)
+=========================
+
+New Features
+------------
+
+- twilio-video.js will now use the Unified Plan SDP format where available.
+  Google Chrome, starting from version 72, has and Safari, starting from version
+  12.1, will enable Unified Plan as the default SDP format. We highly recommend
```

Comment:
12.1 does not have a unified plan. Also, we have already announced support for the unified plan on Safari in 2.0.0-beta6.